tests/unit/onyxTest.ts

```diff
@@ -1454,6 +1454,11 @@ describe('Onyx', () => {
             callback,
         });
 
+        // The snapshot is updated belong to other updates, so we need to set a delay to ensure the connection is done before the snapshot is updated
```

Comment:
This comment is confusing. What do you mean by "other updates" and which onyx "connection" are we talking about?
```suggestion
        // The snapshot being updated belongs to other updates, so we need to set a delay to ensure the connection is done before the snapshot is updated
```

Reply:
Hmm can we use `waitForPromisesToResolve` instead? Why do we need the 100ms timeout instead of just resolving the promise?

Reply:
Nice! Thanks for updating!